eyeshade/worker.js

```diff
@@ -0,0 +1,25 @@
+const dotenv = require('dotenv')
+const utils = require('bat-utils')
+
+const config = require('../config.js')
+
+const reportWorker = require('./workers/reports')
+const {
+  extras,
+  Runtime
+} = utils
+
+dotenv.config()
+
+if (!process.env.BATUTIL_SPACES) {
+  process.env.BATUTIL_SPACES = '*,-hapi'
+}
+
+Runtime.newrelic.setupNewrelic(config, __filename)
+
+const runtime = new Runtime(config)
+
+// call the report worker initialize
+reportWorker(true, runtime)
```

Comment:
I think this might need to be `reportWorker.initialize` because of how we're exporting in `workers/reports`. it may be easiest if we revert parts of ddea44db9651bd843bb8fb27e233cbd1b33fcc06 to get this going again. I think we also had the worker in the docker-compose which will make it easier to test that it's working

Reply:
agree. closing pr.